security/proto/envoy/config/filter/http/alpn/v2alpha1/config.proto

```diff
@@ -0,0 +1,27 @@
+// Copyright 2018 Istio Authors
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+syntax = "proto3";
+
+// $title: ALPN filter for overriding ALPN for upstream TLS connections.
+
+package istio.envoy.config.filter.http.alpn.v2alpha1;
```

Comment:
why v2alpha1? just v1?

Reply:
Actually I don't know which version I should choose. Just follow the pattern from existing filter, e.g. tcp_cluster_rewrite filter

Reply:
that seems a mistake of that filter/weirdness though, without much context...